README.md

```diff
@@ -14,11 +15,11 @@ and Pereira, J. M. B.<sup>3</sup>
 de Janeiro, RJ, Brazil.*
 
 This paper has been published in 
-[**Geochemistry, Geophysics, Geosystems (G<sup>3</sup>)**](https://doi.org/10.1002/2016GC006329).
+[**Geochemistry, Geophysics, Geosystems (G<sup>3</sup>)**](http://agupubs.onlinelibrary.wiley.com/hub/journal/10.1002/(ISSN)1525-2027).
 
-doi: [10.1002/2016GC006329](https://doi.org/10.1002/2016GC006329)
+doi: [10.1002/2016GC006329](http://onlinelibrary.wiley.com/doi/10.1002/2016GC006329/abstract)
```

Comment:
@andrelreis also don't use the publisher link if you can use the DOI. The purpose of the DOI link is that it never changes. If the publisher changes their website, your link will be broken. For example, if AGU go away from Wiley.